### PR TITLE
Adding Comment model

### LINKED ISSUE #23: Adding comment model
Creating the Comment model as written in the ERD

### LINKED ISSUE #66: Adding tests for comment model

### PR DESCRIPTION
**Pull request by MatanP12**
- Creating Comment model.
- Adding Comment model tests.

fix #66 
fix #23

### CODE REVIEW

File: posts/test_comment.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    def test_delete_comment(self, persist_comment):
+        comment = persist_comment
+        comment.delete()
+        assert comment not in Comment.objects.all()
+
+    # Check if deletion of resume delete the comment from database
+    def test_comment_deletion_after_resume_deletion(self, persist_comment):
+        comment = persist_comment
+        comment.post.delete()
+        assert comment.post not in Resume.objects.all()
```
Comment (yulisuliman) [inline posts/test_comment.py:105]: I think we dont need line 105 because we check the deletion of resume in resume tests

Comment (MatanP12) [reply posts/test_comment.py:105]: You are right that we checked the deletion of resume in resume test, I've wanted to check verify the resume is deleted before verifying the deletion of the comment.

**Review by TomerNewmanPrograms — APPROVED** [review]
Looks good

**Review by AlonShakaroff — DISMISSED** [review]
LGTM

**Review by OmerCS8 — DISMISSED** [review]

File: posts/test_comment.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+                        test test test test test test test test test test test test test test test test test test
+                        test test test test test test test test test test test test test test test test test test test
+                        test test test test test test test test test test test test test test test test test test test
+                    """
+
+# -----------------------------------------------fixtures----------------------------------------------------------
+
+
+@pytest.fixture
+def post_creator():
```
Comment (natifridman) [inline posts/test_comment.py:37]: The creator is the author?

File: posts/test_comment.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    new_comment.post.author.save()
+    new_comment.post.save()
+    new_comment.author.save()
+    new_comment.save()
+    return new_comment
+
+
+@pytest.mark.django_db()
+class TestComment:
+
```
Comment (natifridman) [inline posts/test_comment.py:84]: Remove blank line

File: posts/test_comment.py (hunk 2 above)
Comment (MatanP12) [reply posts/test_comment.py:37]: > The creator is the author?

yes I will change the name to post author for readability.

**Review by AlonShakaroff — APPROVED** [review]
LGTM

**Review by TomerNewmanPrograms — DISMISSED** [review]

**Review by OmerCS8 — APPROVED** [review]
LGTM

File: posts/test_comment.py (hunk 2 above)
Comment (natifridman) [reply posts/test_comment.py]: Looks like there are still variables and functions with `creator` in their name

**Review by yulisuliman — APPROVED** [review]

File: posts/admin.py [hunk 4]
```diff
@@ -1,6 +1,8 @@
 from django.contrib import admin
 from .models import Resume
 from .models import Rating
+from .models import Comment
```
Comment (abiton1) [inline posts/admin.py:4]:
```suggestion
from .models import Resume, Rating, Comment
```

File: posts/models.py [hunk 5]
```diff
@@ -4,7 +4,7 @@
 from django.core.validators import FileExtensionValidator
 from django.core.validators import MaxValueValidator
 from django.core.validators import MinValueValidator
-
+from django.core.validators import MaxLengthValidator
```
Comment (abiton1) [inline posts/models.py:7]:
```suggestion
from django.core.validators import (
    FileExtensionValidator,
    MaxValueValidator,
    MinValueValidator,
    MaxLengthValidator,
)
```

**Review by yulisuliman — DISMISSED** [review]

**Review by TomerNewmanPrograms — APPROVED** [review]

**Review by natifridman — DISMISSED** [review]

**Review by OmerCS8 — APPROVED** [review]
LGTM

**Review by TomerNewmanPrograms — APPROVED** [review]

**Review by natifridman — APPROVED** [review]

**Pull request merged**